roles/mesos/tasks/main.yml

```diff
@@ -22,12 +22,3 @@
   with_items: groups['mesos-dns']
   notify: Regenerate /etc/resolv.conf
 
-# The Mesos package installs both the master and agent. Make the Mesos services
```

Comment:
it's moved into Debian and Redhat branches. I noticed that Redhat works in a different way with services.